src/customTheme.ts

```diff
@@ -46,7 +46,9 @@ const customTheme = extendTheme({
     secondary: blueColor,
   },
   fonts: {
-    heading: '"Open Sans", sans-serif;',
+    heading: '"Inter", sans-serif;',
+    body: '"Inter", sans-serif;',
+    mono: '"Inter", sans-serif;',
```

Comment:
This should get us using the _Inter_ for most things...

You said this wasn't a complete replacement of the Chakra UI default font? So we'll probably need to do more work later. But for now it's a good start, I think?

Reply:
This covers all font tokens from Chakra! Unless we want to add more tokens for fonts (I don't think so right?) this is all we need to do.